lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -992,8 +1008,17 @@ Error OpenCLFunction::execute(ExecutionContext *context) {
       // This is a naive implementation that parallelizes using three dims:
       // the X and the Y in the output filter.
       cl_kernel kernel = createKernel(kernelName, program);
-      setKernelArg(kernel, 0, deviceBuffer);
-      auto numArgs = setKernelArgsForBuffers(kernel, I, 1, runtimeBundle_);
+      unsigned numArgs = 0;
+      setKernelArg(kernel, numArgs++, deviceBuffer);
```

Comment:
In this case it looks like it's because I made the deviceBuffer the first argument again. I actually had a lot of trouble making this kernel work correctly and vaguely remember this being the only thing that worked. Will look into it.

Reply:
I've added a comment about this but basically if you remove the first `void*` arg from this kernel it doesn't compile. Why? No idea it should be fine, and all the other kernels were. This is a compromise.